# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296995427 18000
#      Sun Feb 06 07:30:27 2011 -0500
# Branch release-3-4-x
# Node ID c1ea9b90a2722d3813608e30d6b5e7cf1741d77f
# Parent  c468c5b902b3c82ac338bb1e239a7b35f09b38c6
Added tag ss-3-3-92 for changeset c468c5b902b3

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -47,8 +47,9 @@ 74be4b7273e43952e1afcd9ad979d35667c7f82e
 79a56d0a6a0d693ebf461c456951b20dfbcde140 ss-3-3-51
 229675bb7647f7e7e499bcb5acacc8eabe7f3697 ss-3-3-52
 6c69a7c39039bccd5a2fe481fcc7f896c82b160b ss-3-3-53
 3cbc0d77db48aec32bcb202d09a036d2cb9cc3b9 ss-3-3-53
 bd2643f0ce57d070963bedd48857405f6924aa85 ss-3-3-54
 695141f1c05cf1b240592bdd18e7a1503bb2a539 ss-3-3-55
 6c54ad0fde0495cb6972d2db1dc96f25e570f9c0 ss-3-3-90
 b10ea6efdc58565109152b435ac7b9b1a7697223 ss-3-3-91
+c468c5b902b3c82ac338bb1e239a7b35f09b38c6 ss-3-3-92
